src/Git.hs

```diff
@@ -410,6 +432,20 @@ runGit userConfig repoDir operation =
 
     DeleteTag t cont -> cont <$ callGitInRepo ["tag", "-d", refSpec t]
 
+    CheckOrphanFixups sha branch cont -> do
+      result <- let branch' = refSpec branch
+                    sha' = refSpec sha 
+                in callGitInRepo ["log", Text.unpack $ format "{}..{}" [branch',sha'], "--format='%s'"]
+      case result of
+        Left (code, message) -> do
+          logWarnN $ format "git log failed with code {}: {}" (show code, message)
+          pure $ cont False
+        Right logResponse -> do
+          let anyOrphanFixups = any (\x -> "'fixup!" `Text.isPrefixOf` x) $ Text.lines logResponse 
```

Comment:
There's an apostrophe in the `"'fixup!"` string, which probably shouldn't be there.

Reply:
There should be an apostrophe, actually. In our case, git log returns a list of lines enclosed in apostrophes. Either we need to filter these, or having an apostrophe in the prefix check is a necessity.